pkg/server/home_workspaces.go

```diff
@@ -517,12 +527,19 @@ func tryToCreate(h *homeWorkspaceHandler, ctx context.Context, user kuser.Info,
 	parent, name := logicalClusterName.Split()
 	ws := &tenancyv1alpha1.ClusterWorkspace{
 		ObjectMeta: metav1.ObjectMeta{
-			Name: name,
+			Name:        name,
+			Annotations: map[string]string{logicalcluster.AnnotationKey: parent.String()},
 		},
 		Spec: tenancyv1alpha1.ClusterWorkspaceSpec{
 			Type: tenancyv1alpha1.ClusterWorkspaceTypeReference{Path: tenancyv1alpha1.RootCluster.String(), Name: workspaceType},
 		},
 	}
+	logger := logging.WithObject(klog.FromContext(ctx).WithValues(
+		"action", "tryToCreate",
+		"parent", parent.String(),
+		"clusterName", logicalClusterName,
```

Comment:
is clusterName needed or won't it come from ws?

Reply:
I don't know, this flow was hard to follow. I was hoping we could clean it up after if it was to add redundant information.